utilities/test_suite/HIP/Tensor_hip.cpp

```diff
@@ -500,6 +500,25 @@ int main(int argc, char **argv)
 
                 break;
             }
+            case 5:
+            {
+                testCaseName = "pixelate";
+
+                // Note: The 'pixelationIntensity' variable controls how much pixelation is applied to images. 
+                // For images smaller than 100x100 pixels, use 4. 
+                // For images between 100x100 and 300x300 pixels, use 8. 
+                // For images between 300x300 and 1920x1080 pixels, use 10. 
+                // For 4k images, use 12 for better results.
+                Rpp32s pixelationIntensity = 12;
```

Comment:
The default images used is 50x50, 100x100, 150x150 for the test suite
Here the default value used for pixelationIntensity is 12, but ideally as per the description it should be 4 for the above image sizes

What is the reason for keeping this value as 12 by default?

Reply:
changed to 8